src/recordlinker/models/base.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import create_engine
+from sqlalchemy import orm
+from sqlalchemy import types as sqltypes
+
+from recordlinker.config import settings
+
+
+class Base(orm.DeclarativeBase):
+    pass
+
+
+def get_session() -> orm.Session:
+    """
+    Creates a new session to the MPI database and returns it.
+    """
+    engine = create_engine(
+        url=settings.db_uri,
+        pool_size=settings.connection_pool_size,
+        max_overflow=settings.connection_pool_max_overflow,
+    )
+    Base.metadata.create_all(engine)
+    return orm.Session(engine)
+
+
+def get_bigint_pk():
+    """
+    Most databases support auto-incrementing primary keys using BIGINT, however SQLite
+    does not support it.  Thus for the SQLite dialect, we need to use INTEGER instead.
+    """
+    return sqltypes.BigInteger().with_variant(sqltypes.INTEGER, "sqlite")
```

Comment:
I don't know how large these tables will become, but I've been bitten by the INTERGER primary key issue one too many times with large tables.  In most databases (sqlite not included), integers max out at just over 2 billion (4 bytes of values).  Now that seems like a lot, but if we're creating ~12 blocking rows for every patient record that gives you just ~170 million patient records before we run into issues.  Thats a lot of data, and will likely take STLTs years to get there, but its easier to make this change now rather than deal with it a year or two from now.